clients/go/gengo.go

```diff
@@ -111,6 +113,47 @@ func NewFromDiscovery() (*WagClient, error) {
 	return New(url), nil
 }
 
+func newCacheHitCounter(cache httpcache.Cache, basePath string, l logger.KayveeLogger) *cacheHitCounter {
+	chc := &cacheHitCounter{Cache: cache, basePath: basePath}
+	go chc.log(l)
+	return chc
+}
+
+type cacheHitCounter struct {
+	httpcache.Cache
+	hits     int64
+	misses   int64
```

Comment:
probably not worth it yet, but i could see us wanting to know this on a per-endpoint basis